build/Makefile

```diff
@@ -42,6 +42,7 @@ REGISTRY ?= $(release_registry)
 KUBECONFIG ?= ~/.kube/config
 # The (gcloud) test cluster that is being worked against
 GCP_CLUSTER_NAME ?= test-cluster
+GCP_CLUSTER_LOCATION ?=
 GCP_CLUSTER_ZONE ?= us-west1-c
```

Comment:
I think we could keep the Makefile/.mk changes a little less chatty by just collapsing to `GCP_CLUSTER_LOCATION` here:

```
GCP_CLUSTER_ZONE ?= us-west1-c
GCP_CLUSTER_LOCATION ?= $(GCP_CLUSTER_ZONE)
```

That would let you just use `LOCATION` in build/includes/google-cloud.mk and remove the conditionals, since `GCL_CLUSTER_LOCATION` would always be set.

Reply:
This makes sense to me.